pkg/agent/baker.go

```diff
@@ -113,40 +113,28 @@ func (t *TemplateGenerator) getLinuxNodeCSECommand(config *datamodel.NodeBootstr
 
 // getWindowsNodeCSECommand returns Windows node custom script extension execution command
 func (t *TemplateGenerator) getWindowsNodeCSECommand(config *datamodel.NodeBootstrappingConfiguration) string {
-	var userAssignedIdentityClientIDParams string
-	if config.ContainerService.Properties.OrchestratorProfile.KubernetesConfig != nil {
-		isUserAssignedIdentity := config.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.UserAssignedIDEnabled()
-		if isUserAssignedIdentity {
-			userAssignedIdentityClientIDParams = "' -UserAssignedClientID ',reference(variables('userAssignedIDReference'), variables('apiVersionManagedIdentity')).clientId,"
-		}
+	//get parameters
+	parameters := getParameters(config, "", "")
```

Comment:
Will all parameters be generated by agentbaker? How do we validate that parameters generated by agentbaker are as same as that generated by aks-engine?

Reply:
This is what I have do so far to protect the case in https://github.com/Azure/AgentBaker/pull/479 before we comment these codes.
1. we added test scenarios to cover the changes and compared the case command manually
2. I added e2e test  for MSI and SP case to make azure.json is successfully generated